aes/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "aes"
-version = "0.6.0"
-description = "Facade for AES (Rijndael) block ciphers implementations"
+version = "0.7.0-pre"
```

Comment:
Note: as a breaking change, I'd like to refactor/ship this concurrently along with the proposed breaking changes to the `cipher` API here:

https://github.com/RustCrypto/traits/pull/352